src/containers/Desk.js

```diff
@@ -7,47 +7,72 @@ import Write from '../components/Write'
 class Desk extends Component {
   state = {
     letterStack: [],
-    logged_in: null
+    loggedIn: null,
```

Comment:
do we need loggedIn in state here when we are already passing down accountId from App (and therefore already logged in)?

Reply:
You are totally right, we don't need to save it here, good catch!